test/unit/index.js

```diff
@@ -132,7 +138,7 @@ describe('Data shape', function () {
 describe('extractData()', function () {
     it('should always return an object', function () {
         expect(extractData).withArgs().to.not.throwException();
-        expect(extractData()).to.eql({});
+        expect(extractData()).to.be.an('object');
```

Comment:
Better follow the test's spec. whereas before it checked for object-propery equality. With this change calling `extractData()` returns an entry for each locale along with it's `parentLocale` mapping. That's why this test needed to change.